src/queries/products.ts

```diff
@@ -9,7 +9,7 @@ export function useAvailableProducts() {
     "available-products",
     async () => {
       const res = await axios.get<AvailableProduct[]>(
-        `${API_PATHS.bff}/product/available`
+        `${API_PATHS.bff}/products`
```

Comment:
nit: the api gateway URL should be updated for **product**, not for **btt**

Reply:
Hello @vromantsev,
Thanks for your comment.
You are right I did the update in my last commit https://github.com/akiavara/nodejs-aws-shop-react/pull/3/commits

I don't know why I did it in bff...
For the Swagger endpoint I'm not sure to follow your point, because the expected productId is a string but maybe the confusion comes from the fact that I have put "1" and "2" as ID for some products (https://1tmvuptf84.execute-api.eu-west-3.amazonaws.com/dev/products)? That's product I have created manually but it's still string, others products have been created automatically that's the reason why it's real string / uuidv4.

Thanks